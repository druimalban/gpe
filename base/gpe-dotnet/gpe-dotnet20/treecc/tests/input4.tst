// test other tokens
( ) () , = * [ ] ; & ::
"Hello World!"
"012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789"
"open string
""a
'"jkkjkjlkllk'
