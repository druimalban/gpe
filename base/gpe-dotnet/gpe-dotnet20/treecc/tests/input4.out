line 2: (
line 2: )
line 2: (
line 2: )
line 2: ,
line 2: =
line 2: *
line 2: [
line 2: ]
line 2: ;
line 2: &
line 2: ::
line 3: string: "Hello World!"
line 4: string: "012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789"
input4.tst:5: end of line inside string
line 5: string: "open string"
line 6: string: ""
line 6: identifier (len = 1): a
line 7: string: ""jkkjkjlkllk"
